projects/dynamicforms/src/components/upload/upload.component.spec.ts

```diff
@@ -194,6 +208,30 @@ describe('UploadComponent', () => {
     expect(component.fileList.length).toEqual(2);
   });
 
+  it('should not upload file with not supported types', () => {
+    const blob = new Blob([''], { type: 'application/xml' });
+    const mockNotSupportedFile = {
+      target: {
+        files: [<File>blob],
+        multiple: true,
+        accept: 'application/pdf,image/jpeg',
+        value: 'test',
+      },
+    };
```

Comment:
We should maybe move this to the top with the rest of the constants even though it's used only in this test. Gives better overview of the test and keeps test execution cleaner in my opinion.